pxr/usdImaging/usdImaging/implicitSurfaceMeshUtils.cpp

```diff
@@ -436,6 +436,59 @@ UsdImagingGenerateCapsuleMeshPoints(
     return pointsArray;
 }
 
+// Plane -----------------------------------------------------------------------
+
+const PxOsdMeshTopology&
+UsdImagingGetPlaneTopology()
+{
+    static const VtIntArray numVerts{ 4 };
+    static const VtIntArray verts{ 0, 1, 2, 3 };
+    static const PxOsdMeshTopology planeTopo(
+        PxOsdOpenSubdivTokens->bilinear,
+        PxOsdOpenSubdivTokens->rightHanded,
+        numVerts, verts);
+
+    return planeTopo;
+}
+
+VtVec3fArray
+UsdImagingGeneratePlaneMeshPoints(
+    const double width,
+    const double length,
+    const TfToken& axis)
+{
+    int numPoints = 4;
+    std::vector<GfVec3f> points(numPoints);
+
+    if (axis == UsdGeomTokens->x) {
+        points = { GfVec3f( 0.0f,  1.0f * length, 1.0f * width ),
+                   GfVec3f( 0.0f, -1.0f * length, 1.0f * width ),
+                   GfVec3f( 0.0f, -1.0f * length,-1.0f * width ),
+                   GfVec3f( 0.0f,  1.0f * length,-1.0f * width ) };
+    } else if (axis == UsdGeomTokens->y) {
+        points = { GfVec3f(-1.0f * width, 0.0f, 1.0f * length ),
+                   GfVec3f( 1.0f * width, 0.0f, 1.0f * length ),
+                   GfVec3f( 1.0f * width, 0.0f,-1.0f * length ),
+                   GfVec3f(-1.0f * width, 0.0f,-1.0f * length ) };
+    } else {
+        points = { GfVec3f( 1.0f * width, 1.0f * length, 0.0f ),
+                   GfVec3f(-1.0f * width, 1.0f * length, 0.0f ),
+                   GfVec3f(-1.0f * width,-1.0f * length, 0.0f ),
+                   GfVec3f( 1.0f * width,-1.0f * length, 0.0f ) };
```

Comment:
Not sure if this is correct, it seems like this'll create points for a plane that's 2x the desired width and length?